metrics-core/src/main/java/com/codahale/metrics/MetricRegistry.java

```diff
@@ -301,6 +301,19 @@ public boolean isInstance(Metric metric) {
         });
     }
 
+    /**
+     * Return the {@link Gauge} registered under this name; or create and register
+     * a new {@link SettableGauge} if none is registered.
+     *
+     * @param name the name of the metric
+     * @return a new or pre-existing {@link SettableGauge}
+     * @since 4.2
+     */
+    @SuppressWarnings("unchecked")
+    public <T> SettableGauge<T> gauge(String name) {
```

Comment:
Shouldn't line 313 be 
public <T> SettableGauge<T> settableGauge(String name) { ?